extensions/azdata/src/localizedConstants.ts

```diff
@@ -4,33 +4,47 @@
+export const notFoundExistingAzdata = localize('azdata.notFoundExistingAzdata', "Could not find existing azdata installation. Upgrade cannot be performed. Try installing instead");
+
 export const downloadingProgressMb = (currentMb: string, totalMb: string): string => localize('azdata.downloadingProgressMb', "Downloading ({0} / {1} MB)", currentMb, totalMb);
 export const downloadFinished = localize('azdata.downloadFinished', "Download finished");
 export const installingAzdata = localize('azdata.installingAzdata', "Installing azdata...");
 export const upgradingAzdata = localize('azdata.upgradingAzdata', "Upgrading azdata...");
 export const azdataInstalled = localize('azdata.azdataInstalled', "azdata was successfully installed. Restarting Azure Data Studio is required to complete configuration - features will not be activated until this is done.");
 export const azdataUpgraded = localize('azdata.azdataUpgraded', "azdata was successfully upgraded.");
-export const cancel = localize('azdata.cancel', "Cancel");
 export const yes = localize('azdata.yes', "Yes");
 export const no = localize('azdata.no', "No");
+export const always = localize('azdata.always', "Always");
+export const never = localize('azdata.never', "Never");
 export const downloadingTo = (name: string, location: string): string => localize('azdata.downloadingTo', "Downloading {0} to {1}", name, location);
 export const executingCommand = (command: string, args: string[]): string => localize('azdata.executingCommand', "Executing command \"{0} {1}\"", command, args?.join(' '));
-export const stdoutOutput = (stdout: string): string => localize('azdata.stdoutOutput', "stdout : {0}", stdout);
-export const stderrOutput = (stderr: string): string => localize('azdata.stderrOutput', "stderr : {0}", stderr);
-export const checkingLatestAzdataVersion = localize('azdata.checkingLatestAzdataVersion', "Checking for latest version of azdata");
+export const stdoutOutput = (stdout: string): string => localize('azdata.stdoutOutput', "stdout: {0}", stdout);
+export const stderrOutput = (stderr: string): string => localize('azdata.stderrOutput', "stderr: {0}", stderr);
+export const checkingLatestAzdataVersion = localize('azdata.checkingLatestAzdataVersion', "Checking for latest available version of azdata");
 export const gettingTextContentsOfUrl = (url: string): string => localize('azdata.gettingTextContentsOfUrl', "Getting text contents of resource at URL {0}", url);
-export const foundAzdataVersionToUpgradeTo = (version: string): string => localize('azdata.versionForUpgrade', "Found version {0} that azdata-cli can be upgraded to.", version);
+export const foundAzdataVersionToUpgradeTo = (newVersion: string, currentVersion: string): string => localize('azdata.versionForUpgrade', "Found version: {0} that azdata-cli can be upgraded to from current version: {1}.", newVersion, currentVersion);
```

Comment:
Just call it azdata. azdata-cli is specifically the linux version. If we need to refer to the full name we should use `Azure Data CLI`

Reply:
changed